index.ts

```diff
@@ -356,6 +356,33 @@ export const cartExtend = {
                 }
                 commit(types.CART_SET_ITEMS_HASH, getters.getCurrentCartHash) // update the cart hash
               }
+
+              // Re-mapping server items to client items one more time.
+              // Delete client items if server items were deleted - freegift or backend automatically deleted item to quote.
+              await TaskQueue.execute({
```

Comment:
```suggestion
              const task = await TaskQueue.execute({
```

and remove the `then` blockk on row 370